util.go

```diff
@@ -1,6 +1,6 @@
 package nebula_go
 
-func IndexOf(collection []string, element string) int {
+func indexOf(collection []string, element string) int {
```

Comment:
Maybe it’s not appropriate to make this change in the current version because this change involves incompatibility at the user level.